### ISSUE TITLE
How to inference my own test set？

### ISSUE THREAD

**Issue by jplnasa5** [huanghoujing/person-reid-triplet-loss-baseline#23]
I want to use my own photos to inference, how to set up?
Can I migrate to my own test set with a pre-trained model? Or do I have to train my training set first?
thanks

**Comment by huanghoujing** [comment]
Hi, I have added [an example](https://github.com/huanghoujing/person-reid-triplet-loss-baseline#inference) to extract features for a list of images using a trained model.
